utils/cp_utils.py

```diff
@@ -87,7 +87,8 @@ def run_cellprofiler(
                 "--data-file",
                 path_to_loaddata,
             ]
-            subprocess.run(command, capture_output=cellprofiler_output_file, check=True)
+            subprocess.run(command, stdout=cellprofiler_output_file, stderr=cellprofiler_output_file, check=True)
+            print(f"The CellProfiler run has been completed with {pathlib.Path(path_to_loaddata).name}. Please check log file for any errors.")
```

Comment:
I do not think that the print statement will be executed because the `check` is set to `True`. Consider using a `try` and `except` block.

According to the documentation, if `check=True` it will raise `CalledProcessError`. Therefore you can capture the `CalledProcessError` with the `except` block.

Here's an example:
```python
try:
    subprocess.run(command, stdout=cellprofiler_output_file, stderr=cellprofiler_output_file, check=True)
except subprocess.CalledProcessError:
    print(f"The CellProfiler run has been completed with {pathlib.Path(path_to_loaddata).name}. Please check log file for any errors.")
```

Let me know if you have any questions about this suggestion :)!

Reply:
It shows in the notebook that it was printed out: 
![image](https://user-images.githubusercontent.com/107513215/229843261-502a471b-1ab8-470b-9f00-ede8dbaa2628.png)
As well, the statement isn't meant to be an error, just to remind them to check the log file to see if there were any issues in the file like in some pipelines it will give an error that an image is low contrast.
I will keep the in mind if anything changes and the messages I want printed stop printing and try this method as I can see this would work!

Reply:
I have also resolved all comments below relating this one. I think that this would be good to have a discussion about in person since I continue to use these functions in the next module/PR.